Meta/build-image-qemu.sh

```diff
@@ -60,10 +60,10 @@ SCRIPT_DIR="$(dirname "${0}")"
 # Prepend the toolchain qemu directory so we pick up QEMU from there
 PATH="$SCRIPT_DIR/../Toolchain/Local/qemu/bin:$PATH"
 
-# Also prepend the i686 toolchain directory because that's where most
+# Also prepend the x86_64 toolchain directory because that's where most
 # people will have their QEMU binaries if they built them before the
 # directory was changed to Toolchain/Local/qemu.
-PATH="$SCRIPT_DIR/../Toolchain/Local/i686/bin:$PATH"
+PATH="$SCRIPT_DIR/../Toolchain/Local/x86_64/bin:$PATH"
```

Comment:
I think we can actually remove this. Surely no one is still using serenity-build qemu 5.x :)